Makefile

```diff
@@ -69,6 +69,7 @@ deps-test:
 
 # (Re)install the tool
 install: spec
+	$(PIP) install -U "pip>=9.0.0" wheel
```

Comment:
Your changelog says:
> pip 19.0 or newer is required for manylinux2010 platform tag support.
> pip 19.3 or newer is required for manylinux2014 platform tag support.

Yet here you now only require `pip>=9.0`? (And I believe you should use either `-U` or a minimal version.)

Reply:
manylinux2014 seems to be still rarely required. For tensorflow only manylinux2010 is needed.

Reply:
But still, you meant `pip>=19.0.0` not `pip>=9.0.0`, right?

Reply:
My mistake, sorry. I fixed it now.